recipes/arduinojson/all/conanfile.py

```diff
@@ -16,12 +17,29 @@ class ArduinojsonConan(ConanFile):
     topics = ("json", "arduino", "iot", "embedded", "esp8266")
     license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
+    package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
```

Comment:
Are you sure? Some C++11 features would require at least Visual Studio 2019 here?

Reply:
This was a quick workaround to get the C++11, but your pull request has nicer way.